app.py

```diff
@@ -1,35 +1,35 @@
 from flask import Flask, redirect, url_for, request, render_template, Response
-from flask_mail import Mail, Message
 import os
 import json
 import requests
 from multiprocessing import Pool
+import sendgrid
+from sendgrid.helpers.mail import *
 
 app = Flask(__name__)
 
 
 def send_mail(form_msg, form_name, form_email):
-    print("inside async")
-    app.config['MAIL_SERVER'] = 'smtp.gmail.com'
-    app.config['MAIL_PORT'] = 587  # 465 for SSL
-    app.config['MAIL_USE_TLS'] = True
-    app.config['MAIL_USE_SSL'] = False  # True
-    app.config['MAIL_USERNAME'] = 'dibyadasiscool@gmail.com'
-    app.config['MAIL_PASSWORD'] = 'samsunghp'
-    print("all config done")
-    mail = Mail(app)
-    print("init mail app")
-    form_data = form_msg  # request.form['msg']
-    print("message taken")
-    msg = Message('Query', sender='dibyadasiscool@gmail.com',
-                  recipients=['dibyadas998@gmail.com'])
-    print("message init")
-    msg.body = "Query sent by:- " + form_name + "\n" + \
-        form_data + "\n" + "Email-ID is:- " + form_email
-    print("body made")
-    print("going to send the msg")
-    mail.send(msg)
-    print("sent")
+
+    sg = sendgrid.SendGridAPIClient(apikey=os.environ['SENDGRID_API_KEY'])
+    # print("init")
+    from_email = Email(form_email)
+    # print("email init")
+    to_email = Email(os.environ['KOSS_EMAIL'])
+    # print("email init2")
+    subject = "Query"
+    # print("subject")
+    content = Content("text/plain", "Query sent by:- " + form_name + "\n\n\n" +
+                      "Message:- " + form_msg + "\n")
+    # print("content")
+    mail = Mail(from_email=from_email, subject=subject,
+                to_email=to_email, content=content)
+    # print("mail init")
+    try:
+        response = sg.client.mail.send.post(request_body=mail.get())
+        # print("sent")
+    except urllib.HTTPError:
+        print("not sent")
```

Comment:
Add another except to catch all other exceptions and notify user when the mail is not sent